proj/src/main/webapp/signup.js

```diff
@@ -96,7 +116,7 @@ function meetRequirements(password){
 
 function passwordLength(password){
     let alertObj = document.getElementById("pwLength");
-    return styleAlert(alertObj,password.length > 7);
+    return styleAlert(alertObj,password.length > 5);
```

Comment:
Can you also update the failing test case.

Reply:
will do